File: app/src/main/java/com/scatl/uestcbbs/module/message/view/PrivateMsgView.java
package com.scatl.uestcbbs.module.message.view;

import com.scatl.uestcbbs.entity.PrivateMsgBean;

/**
 * author: sca_tl
 * description:
 * date: 2020/1/29 16:31
 */
public interface PrivateMsgView {
    void onGetPrivateMsgSuccess(PrivateMsgBean privateMsgBean);
    void onGetPrivateMsgError(String msg);
}

File: app/src/main/java/com/scatl/uestcbbs/module/message/view/PrivateMsgView.kt
package com.scatl.uestcbbs.module.message.view

import com.scatl.uestcbbs.base.BaseView
import com.scatl.uestcbbs.entity.PrivateMsgBean

/**
 * Created by sca_tl at 2023/3/16 11:31
 */
interface PrivateMsgView: BaseView {
    fun onGetPrivateMsgSuccess(privateMsgBean: PrivateMsgBean)
    fun onGetPrivateMsgError(msg: String?)
    fun onDeletePrivateMsgSuccess(msg: String?, position: Int)
    fun onDeletePrivateMsgError(msg: String?)
}